isotovideo

```diff
@@ -296,6 +296,9 @@ $testfd->write("GO\n");
 my $interactive = 0;
 my $needinput   = 0;
 
+# screen checking interval in s (float)
+my $interval = 1.;
```

Comment:
I don't like this global variable. This is internal to one function and shouldn't be global